packages/e2e-tests/tests/logpoints-04.test.ts

```diff
@@ -28,8 +28,6 @@ test(`should display exceptions in the console`, async ({ page }) => {
   await seekToConsoleMessage(page, messages.first());
   await waitForFrameTimeline(page, "100%");
 
-  messages = await findConsoleMessage(page, "number: 10", "exception");
-  await seekToConsoleMessage(page, messages.first());
```

Comment:
Ooo, nice catch! Although I _am_ curious why this reliably worked locally but not on CI.

Reply:
My guess would be that we had a bug in our test helper so that `await seekToConsoleMessage(...)` didn't really wait for the app to jump to the new point and so the evaluation was done on the previous point.